README.md

```diff
@@ -18,33 +18,41 @@ The full list of [contributors](http://github.com/GPflow/GPflow/graphs/contribut
   Ghahramani. GPflow is an open source project so if you feel you have some relevant skills and are interested in contributing then please do contact us.
 
 
-## Announcement:
+## What does GPflow do?
 
-_**The development of the GPflow 2.0 moved to the `develop` branch. The old GPflow 1.0 branch has been transferred to the `develop-1.0`. We stop supporting GPflow 1.0 in terms of new feature and we will accept only bug fixes into `develop-1.0` branch.**_
+GPflow implements modern Gaussian process inference for composable kernels and likelihoods. The [online documentation (develop)](http://gpflow.readthedocs.io/en/develop/)/[(master)](http://gpflow.readthedocs.io/en/master/) contains more details.
 
+GPflow 2.0 uses [TensorFlow 2.0](http://www.tensorflow.org) for running computations, which allows fast execution on GPUs, and uses Python ≥ 3.6.
 
-## What does GPflow do?
 
-GPflow implements modern Gaussian process inference for composable kernels and likelihoods. The [online documentation (develop)](http://gpflow.readthedocs.io/en/develop/)/[(master)](http://gpflow.readthedocs.io/en/master/) contains more details. The interface follows on from [GPy](http://github.com/sheffieldml/gpy), and the docs have further [discussion of the comparison](http://gpflow.readthedocs.io/en/develop/intro.html#what-s-the-difference-between-gpy-and-gpflow).
+## Install GPflow
 
-GPflow 2.0 uses [TensorFlow 2.0](http://www.tensorflow.org) for running computations, which allows fast execution on GPUs, and uses Python **3.6**.
+- From source
 
+  With the release of _TensorFlow 2.0_ and _Tensorflow Probability_ 0.8, you should
+  only need to run
 
-## Install GPflow 2.0 alpha version
+  ```bash
+  pip install -e .
+  ```
 
-With the release of `tensorflow` 2.0 and `tensorflow-probability` 0.8, you should
-only need to run
+  in a check-out of the `develop` branch of the GPflow github repository.
 
-```bash
-pip install -e .
-```
+- Using `pip`
+
+  ```bash
+  pip install gpflow
+  ```
+
+
+## Getting Started with GPflow 2.0
 
-in a check-out of the `develop` branch of the GPflow github repository.
+There is an ["Intro to GPflow 2.0"](https://github.com/GPflow/GPflow/blob/develop/doc/source/notebooks/intro_to_gpflow2.ipynb) Jupyter notebook. Check it out for details.
 
+- **GPflow 1.0**
 
-## Getting Started
+  *We stop the support for GPflow based on TensorFlow 1.0. We accept bug fixes only to GPflow   1.0 in **develop-1.0** branch. Latest available release is [v1.5.1](https://github.com/GPflow/GPflow/releases/tag/v1.5.1).*
```

Comment:
```suggestion
  *We have stopped development and support for GPflow based on TensorFlow 1.0. We now accept only bug fixes to GPflow 1.0 in the **develop-1.0** branch. The latest available release is [v1.5.1](https://github.com/GPflow/GPflow/releases/tag/v1.5.1).*
```

Reply:
tempted to just commit this, but you should do it. happy to accept with this done.